util/envDynawo.sh

```diff
@@ -291,46 +350,95 @@ set_environment() {
   fi
 
   # OpenModelica config
-  export_var_env OPENMODELICA_VERSION=1_9_4
+  export_var_env_force OPENMODELICA_VERSION=1_9_4
+  export_var_env_force MODELICA_LIB=3.2.2
   export_var_env SRC_OPENMODELICA=UNDEFINED
   export_var_env INSTALL_OPENMODELICA=UNDEFINED
 
+  # JQuery config
+  export_var_env JQUERY_DOWNLOAD_URL=https://github.com/jquery/jquery/archive
+  export_var_env FLOT_DOWNLOAD_URL=https://github.com/flot/flot/archive
+
   # Export library path, path and other standard environment variables
-  set_standardEnvironmentVariables
+  set_standard_environment_variables
 
   set_commit_hook
 
   set_cpplint
 }
 
-set_standardEnvironmentVariables() {
-  export LD_LIBRARY_PATH=$NICSLU_INSTALL_DIR/lib:$SUITESPARSE_INSTALL_DIR/lib:$SUNDIALS_INSTALL_DIR/lib64:$LIBZIP_HOME/lib:$LIBXML_HOME/lib:$LIBIIDM_HOME/lib:$DYNAWO_INSTALL_DIR/lib:$ADEPT_INSTALL_DIR/lib:$XERCESC_INSTALL_DIR/lib:$LD_LIBRARY_PATH
+ld_library_path_remove() {
+  export LD_LIBRARY_PATH=`echo -n $LD_LIBRARY_PATH | awk -v RS=: -v ORS=: '$0 != "'$1'"' | sed 's/:$//'`;
+}
 
-  if [ $LIBARCHIVE_HOME_DEFAULT != true ]; then
-    export LD_LIBRARY_PATH=$LIBARCHIVE_HOME/lib:$LD_LIBRARY_PATH
+ld_library_path_prepend() {
+  if [ ! -z "$LD_LIBRARY_PATH" ]; then
+    ld_library_path_remove $1
```

Comment:
If I understand well we remove the dynawo environment paths and then directly put them back. Shouldn't we let the LD_LIBRARY_PATH as it is if it was already set?

Reply:
The function is generic, it is to avoid prepending multiple times the same path